scaffolder-templates/build/.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+stages:
+  - update
+
+update_pipelinerun:
+  tags:
+    - shared-podman # Change to whatever tag your Gitlab runner is set to 
+  stage: update
+  image: ubuntu:latest
+  script: |
+    apt-get update && apt-get install -y git
+    apt-get install apt-transport-https ca-certificates -y 
+    update-ca-certificates
```

Comment:
are those necessary in the end? Did you try to remove them see if that fails or works?

Reply:
The CA-Certs are needed, 
"apt-get install apt-transport-https" - I'm not sure, haven't tested without it

Reply:
OK, apt-get install apt-transport-https and the CA certs can be removed. Git still has to be installed